src/System.Diagnostics.Process/src/System/Diagnostics/Process.Win32.cs

```diff
@@ -2,6 +2,8 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System;
```

Comment:
Nit: System doesn't need to be imported - the namespace is already a child of System.